test/SafeProtocolMediator.spec.ts

```diff
@@ -0,0 +1,496 @@
+import hre from "hardhat";
+import { loadFixture } from "@nomicfoundation/hardhat-toolbox/network-helpers";
+import { expect } from "chai";
+import { SignerWithAddress } from "@nomicfoundation/hardhat-ethers/signers";
+import { TestModule } from "../typechain-types";
+import { ZeroAddress } from "ethers";
+
+describe("SafeProtocolMediator", async () => {
+    let deployer: SignerWithAddress, owner: SignerWithAddress, user1: SignerWithAddress, user2: SignerWithAddress;
+    const SENTINEL_MODULES = "0x0000000000000000000000000000000000000001";
```

Comment:
this will be definitely used outside this test suite, worth exporting to constants